.travis.yml

```diff
@@ -3,22 +3,23 @@ cache:
   directories:
     - $HOME/.cache/pip
 language: python
+python:
+    - "2.6"
+    - "2.7"
+    - "3.3"
+    - "3.4"
+    - "3.5"
+    - "3.6"
+    - "pypy"
+    - "pypy3"
+# Drop support until travis fixes python3 pypy interpreter path issue
+# https://github.com/travis-ci/travis-ci/issues/6304#ref-issue-177592739
```

Comment:
Are these comments still correct if `pypy3` is now in the list?

This seems more relevant to the tox configuration, which doesn't include `pypy3`.

Reply:
Looks like the comment is outdated.